### DISCUSSION TITLE
403 error while exporting donations: too long URL

### DISCUSSION THREAD

**Discussion by Genevieve-K** [impress-org/givewp#7419] [Developer Q&A]
As an administrator, I would like to export donations. For several sites, I get a 403 error. After investigation, I understand that it's due a URL containing too many characters. When we export all fields, the Javascripts `window.location = t.url` failed with an 403 error. If I uncheck some fields, the URL is shorter and the donation file is downloaded.

A idea: the label `give_give_donations_export_option` is used many times. It's difficult to change it because it is used by other plugins.Maybe change it to a short name before creating the URL?

I also have an error downloading donations. I suggest the PR [7417](https://github.com/impress-org/givewp/pull/7417).

**Comment by kjohnson** [comment]
Interesting that the URL is too long. If I understand correctly, that would be a browser limitation issue, but I would expect the limit to be rather large.

How long is the URL?

Also, have you tried it in another browser?

**Comment by Genevieve-K** [comment]
Sorry, I didn't answer. If I get this problem again, I'll come back.

**Reply by JasonTheAdams** [reply]
No worries! Hopefully you don't run into it again! 😄
